templates/web/common/includes/display_field.tt.html

```diff
@@ -1,22 +1,8 @@
 <!-- start templates/[% template.name %] -->
```

Comment:
Personally, I would delete the carriage return of lines 2 and 7, and add one at the beginning of first line and the end of last line.

```
<!-- start templates/web/common/includes/display_field.tt.html -->
<p id="field_packaging">
    <span class="field">Packaging: </span>
    <span class="field_value" id="field_packaging_value"><a href="/packaging/bocal">Bocal</a>, <a href="/packaging/verre">Verre</a>, <a href="/packaging/couvercle">Couvercle</a>, <a href="/packaging/metal">Métal</a>, <a href="/packaging/glas">Glas</a>, <a href="/packaging/97">97</a></span>
</p>
<!-- end templates/web/common/includes/display_field.tt.html -->

<!-- start templates/web/common/includes/display_field.tt.html -->
<p id="field_brands">
    <span class="field">Brands: </span>
    <span class="field_value" id="field_brands_value"><a itemprop="brand" href="/brand/lidl">Lidl</a>, <a itemprop="brand" href="/brand/maribel">Maribel</a></span>
</p>
<!-- end templates/web/common/includes/display_field.tt.html -->

```